Xamarin.Forms.Controls.Issues/Xamarin.Forms.Controls.Issues.Shared/Issue8881.xaml.cs

```diff
@@ -0,0 +1,25 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+using Xamarin.Forms;
+using Xamarin.Forms.Xaml;
+
+namespace Xamarin.Forms.Controls.Issues
+{
+	public partial class Issue8881 : TestShell
```

Comment:
@pictos can you change this over to using unit tests?

We should be able to write unit tests to call `IShellController.GenerateFlyoutGrouping()` to verify the fix and that all variations are working. For example when we set `FlyoutDisplayOptions == FlyoutDisplayOptions.AsMultipleItems`

Reply:
@PureWeen After changed it to a unit test, do you want to delete this file?

Reply:
Yes please!